packages/ibc-utils/src/neutron.rs

```diff
@@ -80,12 +80,11 @@ pub fn ibc_send_message(
             receiver: to.clone(),
             token: Some(coin),
             timeout_height: None,
-            timeout_timestamp: timeout_timestamp.unwrap_or(
-                env.block
-                    .time
-                    .plus_seconds(DEFAULT_TIMEOUT_TIMESTAMP)
-                    .nanos(),
-            ),
+            timeout_timestamp: env
+                .block
+                .time
+                .plus_seconds(timeout_seconds.unwrap_or(DEFAULT_TIMEOUT_TIMESTAMP))
+                .nanos(),
```

Comment:
I think this is incorrect and why an overflow is happening right? Should this be nanos? In the generic transfer you are not converting to nanos

Reply:
nevermind checked the spec and it seems it's correctly in nanos